src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\Filesystem\Tests;
 
+use Symfony\Component\Finder\Finder;
```

Comment:
I ended up using the finder in the Filesystem component tests, is that OK? Should I add it to require-dev of the component I guess?

Reply:
avoiding the finder dependency by using a native iterator would be best